### ISSUE TITLE
Hyper-parameters of Lab-3-3

### ISSUE THREAD

**Issue by Peter-Chuang** [2017-fall-DL-training-program/VAE-GAN-and-VAE-GAN#8]
Dear TA,

In the Lab, you said the learning rate of discriminator should be smaller than generator.
The learning rate of G is 3e-4. Suggested learning rate of D is 3e-5.
But in the latest version of Lab-3-3 PDF document, this is not mentioned.
So, should we set the learning rate of D to 3e-4 or 3e-5?
And should we set the Beta of VAE cost?

Peter

**Comment by hui-po-wang** [comment]
Corrected.

**Comment by Peter-Chuang** [comment]
Thanks!
